templates/ssmtp.conf.j2

```diff
@@ -14,6 +14,7 @@ UseTLS=Yes
 {% endif %}
 {% if sansible_ssmtp_use_start_tls %}
 UseSTARTTLS=Yes
+TLS_CA_File=/etc/pki/tls/certs/ca-bundle.crt
```

Comment:
I don't think this will work on Debian/Ubuntu.

Reply:
The path is `/etc/ssl/certs/ca-certificates.crt` on debian/ubuntu.